app/Services/Manager.php

```diff
@@ -495,4 +495,27 @@ public function addUserToModel($model, $id, $user)
 
         return true;
     }
+
+    /**
+     * Fires off an event to send an email a user.
+     *
+     * @param  Gladiator\Models\User     $user
+     * @param  Gladiator\Models\Contest  $contest
+     */
+    public function sendEmail($user, $contest)
```

Comment:
Maybe you pass in a `$parameters` var that is an array like, 

```php
[
  'type' => 'welcome',
  'key' => '1',
]
```
and you use those params in the `where` statement when grabbing the `$message`